.golangci.yml

```
@@ -54,6 +54,14 @@ linters:
             - pkg: crypto/md5
             - pkg: crypto/sha1
             - pkg: crypto/**/pkix
+        semconv:
+          list-mode: lax
+          deny:
+            - pkg: go.opentelemetry.io/otel/semconv
+              desc: Use go.opentelemetry.io/otel/semconv/v1.37.0 instead. If a newer semconv version has been released, update the depguard rule.
```

Same comment as I made on collector.

```suggestion
              desc: Use go.opentelemetry.io/otel/semconv/v1.38.0 instead. When a newer semconv version is released, it must be added to the depguard rule.
```
Or

```suggestion
              desc: Use go.opentelemetry.io/otel/semconv/v1.38.0 instead. If a newer semconv version has been released, update the depguard rule.
```